tests/core/conversations_worker/integration/broadcast_health_check.py

```
@@ -0,0 +1,232 @@
+import sys
+import threading
+import time
+from datetime import datetime, timezone
+
+from realtime._async.client import AsyncRealtimeClient
+from supabase import create_client
+from supabase.lib.client_options import ClientOptions
+
+from holmes.core.conversations_worker.realtime_manager import broadcast_submit_topic
+
+_log_file = os.environ.get("BROADCAST_HEALTH_LOG", "/tmp/broadcast-health.log")
```

Ruff (0.15.10)

[error] Probable insecure usage of temporary file or directory: "/tmp/broadcast-health.log"

(S108)